### ISSUE TITLE
JS fiddle?

### ISSUE THREAD

**Issue by tabrez500** [serebrov/emoji-mart-vue#67]
Hello, I was trying to work around this vue component but just wondering , is there any JS fiddle available for this. As it is a bit confusing to read the docs.
Thanks

**Comment by serebrov** [comment]
As far as I know, there is no js fiddle or something like that (at least, I didn't make any).

There is a demo project under the [docs folder](https://github.com/serebrov/emoji-mart-vue/tree/master/docs) and [live version here](https://serebrov.github.io/emoji-mart-vue/).

Also, I know that docs are quite a mess as I took the original README and extended it here and there, so it might be hard to use. If you have some specific complains or proposals for docs, please let me know, I am considering to rewrite the readme at some point.

**Comment by tabrez500** [comment]
Yeah I mean I have not gone through the whole docs, but the first piece of code I came across had some confusion like: 
```
import data from '../data/all.json'
import { Picker, EmojiIndex } from 'emoji-mart-vue-fast'
let emojiIndex = new EmojiIndex(data)
```
First-line seems to be directed to local files meanwhile the next line is pointing to the npm package.
So I thought it would be better to add a fiddle link.
